### PR TITLE
Feat ci

### PR DESCRIPTION
**Pull request by madarche**
This is ongoing work for #5 

The tests are not passing at all because all the files are not conforming to PEP8. But it's a start on which to talk about.

@gaetano-guerriero should I modify all the files now so that they are PEP8 conformant?

### CODE REVIEW

**Comment by gaetano-guerriero** [comment]
I think we could run with `pep8 || true` to make test passes, and defer the style fixes.

BTW, we could run our test matrix with `tox`, and than in travis simply execute tox.
This way we can locally replicate tests done by travis.

**Comment by madarche** [comment]
OK for tox. I'll set it up a bit later. Thanks

**Comment by gaetano-guerriero** [comment]
@madarche should we merge this as it is, and remove `|| true` when files are fixed ?
Anyway now there merge conflicts

**Comment by madarche** [comment]
@gaetano-guerriero I'll be working on it again in the next hours. I'll keep you informed. Sorry for the delay.

**Comment by gaetano-guerriero** [comment]
:+1: No problem

**Comment by madarche** [comment]
@gaetano-guerriero this is ready for you to review and hopefully to merge :-) This adds simple PEP8 fixes and a valid PEP8 tests in CI, even if there are many ignored PEP8 warnings. But at least it checks for real that no other warning and error types are introduced in the code base.

I'll be working on adding `tox` in another branch ASAP, so that the above modifications are not delayed anymore.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Jack
 
+[![Build Status](https://travis-ci.org/jack-cli-cd-ripper/jack.svg?branch=feat-ci)](https://travis-ci.org/jack-cli-cd-ripper/jack)
```
Comment (gaetano-guerriero) [inline README.md]: Should  we set `branch=master` now ?

**Comment by gaetano-guerriero** [comment]
Wow that was gigantic. I made a comment on build-status icon

**Comment by madarche** [comment]
@gaetano-guerriero I've fixed the branch name for the CI. Well spotted! ;-) Merge it when you want if you find it's OK.

There are still many changes to do to be fully PEP8 compliant. The most part of the commited PEP8 fixes were done through `autopep8` in safe mode. `autopep8 --aggressive` would produce more fixes. But again, the aim here is just to have a minimal safety net that we will increase as time goes by.

**Comment by gaetano-guerriero** [comment]
:+1: I'm merging now. I'm working on `jack` package grouping and I take this as starting point.

**Comment by madarche** [comment]
@gaetano-guerriero cool :-) I'm looking forward to see you work on this :-)

**Comment by madarche** [comment]
@gaetano-guerriero I still need to kickstart the Travis CI. It's just one more commit on a file. I'm doing that. It will be a commit on a text file so not to disturb your work on the code.

**Comment by madarche** [comment]
We now have our green "build passing" badge :-)

**Pull request merged**